ci/run.sh

```diff
@@ -1,14 +1,14 @@
 #!/bin/bash
 set -ex
 
-export PATH=$PATH:$TRAVIS_BUILD_DIR/gradle/bin/
```

Comment:
We need this, kinda, otherwise `gradle` is not guaranteed to be in the path.

It should probably be set after BUILD_DIR is set, and then using this:

```
export PATH="${PATH}:${BUILD_DIR}/gradle/bin"
```

```
+ git clone https://github.com/elastic/elasticsearch.git es_src
Cloning into 'es_src'...
remote: Counting objects: 843097, done.
remote: Compressing objects: 100% (241/241), done.
remote: Total 843097 (delta 155), reused 104 (delta 37), pack-reused 842789
Receiving objects: 100% (843097/843097), 356.54 MiB | 13.60 MiB/s, done.
Resolving deltas: 100% (464622/464622), done.
Checking connectivity... done.
+ cd es_src
+ gradle :distribution:zip:assemble
ci/run.sh: line 92: gradle: command not found
+ finish
```